CHANGELOG.md

```diff
@@ -40,3 +40,4 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   [cyberark/conjur-openapi-spec#60](https://github.com/cyberark/conjur-openapi-spec/issues/60)
 - Google Cloud Provider authenticate endpoint now included in the spec file. Users can now authenticate with Conjur using GCP.
   [cyberark/conjur-openapi-spec#61](https://github.com/cyberark/conjur-openapi-spec/issues/61)
+- New options to `generate_client` script for greater control over generated output.
```

Comment:
@john-odonnell please make sure to include a link with changelog messages. Maybe it would also be good to highlight here that in this you've added a human-readable Wiki page that's updated nightly to the project?